recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
     - pip
   run:
     - python ==2.7.*|>=3.3
```

Comment:
The dash feedstock was changed not to support 2.7, so these should too.

Reply:
Done in 89f7ed6